tools/generator/Java.Interop.Tools.Generator.Importers/XmlApiImporter.cs

```diff
@@ -331,6 +331,10 @@ public static InterfaceGen CreateInterface (XElement pkg, XElement elem, CodeGen
 					 !options.SupportNestedInterfaceTypes
 			};
 
+			if (elem.Attribute ("skipInvokerMethods")?.Value is string skip)
+				foreach (var m in skip.Split (new char [] { ',', ' ', '\n', '\r' }, StringSplitOptions.RemoveEmptyEntries))
+					iface.SkippedInvokerMethods.Add (m);
```

Comment:
It occurs to me: should we "sanity check" this value, asserting that it contains *one* `.`, *one* `(`, and *one* `)`?  (And maybe even verify that `(…)…` constitutes a valid JNI method signature, according to e.g. `JniMemberSignature.GetParameterCountFromMethodSignature()` + a return type sanity check.)

This request might be a step "too far" for *this* PR -- as opposed to my other comments, which I think are closer to "code hygiene" and code deduplication -- but this thought hadn't occurred to me before.